packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -295,17 +299,20 @@ const Updates = decorate([
                     () => state.installedPackages['xen-orchestra'],
                     'unknown'
                   )}{' '}
-                  {state.installedPackages !== undefined && (
-                    <Button
-                      name='showPackagesList'
-                      onClick={effects.toggleState}
-                      size='small'
-                    >
-                      <Icon icon={state.showPackagesList ? 'minus' : 'plus'} />
-                    </Button>
-                  )}
+                  <Button
+                    name='showDetails'
+                    onClick={effects.toggleState}
+                    size='small'
+                  >
+                    <Icon icon={state.showDetails ? 'minus' : 'plus'} />
+                  </Button>
                 </p>
-                {state.showPackagesList && (
+                {state.showDetails && (
```

Comment:
You can all time show it like the `XO` version. And the toggle will only be for the package list because it's long.

This change is related to this [comment](https://github.com/vatesfr/xen-orchestra/pull/4694#discussion_r353100054). ping @julien-f